docs/_sass/sidebar.scss

```diff
@@ -113,7 +113,7 @@
         font-size: 1.1rem;
         border-radius: 21px;
         background: var(--background-neumorphic);
-        padding: 8px 15px 8px 15px;
+        padding: 0px 15px 8px 15px;
```

Comment:
```suggestion
        padding: 8px 15px 8px 15px;
```

Reply:
imo, let's keep the top padding as is, setting it to 0 results in:
<img width="258" alt="Screenshot 2023-12-21 150114" src="https://github.com/meshery/meshery/assets/110674407/b3791f6d-da19-42b9-a018-41f65877ea0e">